LEGO1/mxentity.cpp

```diff
@@ -9,4 +9,12 @@ MxEntity::MxEntity()
 // OFFSET: LEGO1 0x1000c110
 MxEntity::~MxEntity()
 {
+}
+
+// OFFSET: LEGO1 0x10001070
+MxResult MxEntity::SetEntityId(MxS32 p_id, MxAtomId *p_atom)
+{
+  this->m_mxEntityId = p_id;
+  // FIXME: MxAtomId operator call, probably it will be return whatever the comparison is for comparing this->m_atom to the p_atom
+  return FALSE;
```

Comment:
The MxAtomId `operator=` call is just setting the object with `=`. Most likely that means the param is actually a reference (`&`) rather than a pointer (`*`) - the two are indistinguishable from an asm point of view but this would make more sense code-wise if using `operator=`. Also if it's an MxResult and returns 0, it's almost certainly returning `SUCCESS` not `FALSE`.

```suggestion
MxResult MxEntity::SetEntityId(MxS32 p_id, const MxAtomId &p_atom)
{
  this->m_mxEntityId = p_id;
  this->m_atom = p_atom;
  return SUCCESS;
```